CacheTagsBundle/Annotation/CacheTag/AbstractCacheTag.php

```diff
@@ -0,0 +1,62 @@
+<?php
+
+namespace lbarulski\CacheTagsBundle\Annotation\CacheTag;
+
+use Doctrine\Common\Annotations\Annotation\Required;
+use Doctrine\Common\Annotations\Annotation\Target;
+use lbarulski\CacheTagsBundle\Tag\TagInterface;
+use Sensio\Bundle\FrameworkExtraBundle\Configuration\ConfigurationInterface;
+
+/**
+ * @Annotation
+ * @Target("METHOD")
+ */
+abstract class AbstractCacheTag implements TagInterface, ConfigurationInterface
```

Comment:
here `Plain` is just used to give You the ability to read what was written in the annotation the easy way - treat it like a DTO, nothing more - listener should do the job - we'll talk this through tomorrow :)